### DISCUSSION TITLE
Getting only sporadic updates anmore

### DISCUSSION THREAD

**Discussion by jupppo** [johanmeijer/grott#234] [Q&A]
Hi there,

since yesterday I am getting only sporadic updates form grott (rough once every 30min)
Status says the following (see picture).

Any idea why?
![Clipboard01](https://user-images.githubusercontent.com/69680107/201472826-52222b8b-1842-4b01-b30d-f948385fd123.jpg)

**Comment by robupham** [comment]
Following.

My system is also not working right (using 2.7 beta branch). I think Growatt is having issues with their platform this morning (ShinePhone app won't connect) and I have a theory that when the Growatt cloud platform is broken, Grott also doesn't process data - as I'm not getting data updated in InfluxDB either. Is that correct?

The last data I see is at 10.35 GMT/UTC on the server.growatt.com portal and 10.38 GMT/UTC in my InfluxDB.

**Comment by jupppo** [comment]
Mine is about the same time. Looks like an server issue.
I am now trying to get grottserver to run, but no luck yet....

**Comment by jupppo** [comment]
I stopped grottserver (which doesnt work for me - possible noob error by me)
and entered a different growatt Server IP:

Nov 12 13:49:29 grott grott[5218]:         growattport:                  5279
Nov 12 13:49:29 grott grott[5218]:         growattip:                    47.254.133.160

Now I am getting data again :-)

**Comment by jupppo** [comment]
One wiered thing though: I am getting different time stamps:

Nachricht 9 empfangen auf energy/growatt um 13:57:

{
    "device": "OMCUCDV020",
    "time": "2022-11-12T13:44:20",
    "buffered": "yes",

QoS: 0 - Retain: false
Nachricht 8 empfangen auf energy/growatt um 13:57:

{
    "device": "OMCUCDV020",
    "time": "2022-11-12T13:57:49",
    "buffered": "no",

I assume the shinefy tries to send out the old data to the system

**Comment by johanmeijer** [comment]
Yes that is right buffered = yes means "old" (speichert) data.
